contracts/account/extensions/AccountERC6900.sol

```diff
@@ -0,0 +1,494 @@
+    }
+
+    /// @inheritdoc IERC6900ModularAccount
+    function installValidation(
+        ValidationConfig validationConfig,
+        bytes4[] calldata selectors,
+        bytes calldata installData,
+        bytes[] calldata hooks
+    ) public virtual onlyEntryPointOrSelf {
+        _installValidation(validationConfig, selectors, installData, hooks);
+    }
+
+    /// @inheritdoc IERC6900ModularAccount
+    function uninstallValidation(
+        ModuleEntity validationFunction,
+        bytes calldata uninstallData,
+        bytes[] calldata hookUninstallData
+    ) public virtual onlyEntryPointOrSelf {
+        // _uninstallModule(moduleTypeId, module, deInitData);
+    }
+
+    /// @inheritdoc IERC6900ModularAccount
+    function installExecution(
+        address module,
+        ExecutionManifest memory manifest,
+        bytes calldata installData
+    ) public virtual onlyEntryPointOrSelf {
+        _installExecution(module, manifest, installData);
+    }
+
+    /// @inheritdoc IERC6900ModularAccount
+    function uninstallExecution(
+        address module,
+        ExecutionManifest calldata manifest,
+        bytes calldata uninstallData
+    ) public virtual onlyEntryPointOrSelf {
+        // _uninstallModule(moduleTypeId, module, deInitData);
+    }
+
+    /// @dev Executes a transaction from the entry point or the account itself. See {_execute}.
+    function execute(
+        address target,
+        uint256 value,
+        bytes calldata data
+    ) public payable virtual onlyEntryPointOrSelf returns (bytes memory) {
+        return _execute(target, value, data);
+    }
+
+    function executeBatch(Call[] calldata calls) public payable virtual onlyEntryPointOrSelf returns (bytes[] memory) {
+        bytes[] memory returnedData = new bytes[](calls.length);
+        for (uint256 i = 0; i < calls.length; i++) {
+            returnedData[i] = _execute(calls[i].target, calls[i].value, calls[i].data);
+        }
+        return returnedData;
+    }
+
+    function executeWithRuntimeValidation(
+        bytes calldata data,
+        bytes calldata authorization
+    ) public payable virtual onlyEntryPointOrSelf returns (bytes memory) {
+        ModuleEntity validationModuleEntity = ModuleEntity.wrap(bytes24(authorization[:24]));
+        bytes calldata validationAuth = authorization[24:];
+        IERC6900ValidationModule(validationModuleEntity.module()).validateRuntime(
+            address(this),
+            validationModuleEntity.entityId(),
+            msg.sender,
+            msg.value,
+            data,
+            validationAuth
+        );
+        return _execute(address(this), msg.value, data);
+    }
+
+    /**
+     * @dev Validates a user operation with {_signableUserOpHash} and returns the validation data
+     * if the module specified by the first 20 bytes of the nonce key is installed. Falls back to
+     * {AccountCore-_validateUserOp} otherwise.
```

Comment:
Is the fallback currently implemented?

Reply:
Thank you, indeed fallback was not implemented at all, please see [Init fallback](https://github.com/OpenZeppelin/openzeppelin-community-contracts/pull/80/commits/19df12ef2a73862cfc9eebf04525f02e0a617034) now.

Reply:
Ah, I was referring to the fallback to super mentioned in `_validateUserOp`. Sorry for the confusion.